app/src/pages/authoring/[resourceType]/[id].tsx

```diff
@@ -170,6 +204,28 @@ export default function ResourceAuthoringPage() {
             <ArtifactFieldInput label="name" value={name} setField={setName} />
             <ArtifactFieldInput label="title" value={title} setField={setTitle} />
             <ArtifactFieldInput label="description" value={description} setField={setDescription} />
+            {resourceType === 'Measure' && (
+              <Select
```

Comment:
I did notice that if you select a library who has duplicate options, the duplicates get highlighted as being selected as well. I figure this is mainly a result of the fact that we can have duplicates in the first place, so I don't think it's something you need to fix. Just figured it's worth noting! 
![Screenshot 2023-07-12 at 12 19 58 PM](https://github.com/projecttacoma/measure-repository/assets/98363677/05711098-ab73-4f47-9563-acb7d20d22a5)

Reply:
On the same note, I also noticed that this error message pops up whenever I click the select component for an artifact with duplicate libraries: 
![Screenshot 2023-07-12 at 12 24 45 PM](https://github.com/projecttacoma/measure-repository/assets/98363677/7dfdcb58-1e14-4bf2-8f26-8b49b37cbd7e)
While adding a key to your select component doesn't fix this error message because duplicate libraries would have the same key, I think it would be a good addition for whenever the duplicate problem is gone!

Reply:
Duplicate urls should be addressed in ticket 1344 "Add URL constraint on drafting resources", which pertains to [spec](https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html#draft) "It is an error to create multiple drafts of the same artifact by URL."